_posts/2021-01-05-Jamulus-Sound-Devices.md

```diff
@@ -494,3 +494,5 @@ See above. Not fully verified.
 * **Zoom B3**, ❌ USB amplifier modeling pedal for bass. **Does not support 48 kHz**.
 
 * **Line6 Bass POD**, ❌ USB amplifier modeling pedal for bass. **Does not support 48 kHz**.
+
+* **Yamaha THR5 THR10 Series**, ❌ USB modeling amplifier. **Do not support 48 kHz** (THR30II only is both 44kHz and has a line-out connector).
```

Comment:
If this is _only_ for the THR30II, then this entry should only be for that specific product rather than the series.  Otherwise the explanation should not be specific to the one product.  Also, the other products listed here all appear to be output-only as well and do not mention that.  Either it should be mentioned for all or should remain consistent.
```suggestion
* **Yamaha THR5 THR10 Series**, ❌ USB modeling amplifier. **Do not support 48 kHz**.
```